pio/src/iSpindel.cpp

```diff
@@ -786,60 +786,38 @@ void sleepManager()
   }
 }
 
-void requestTemp()
-{
-  if (!DSreqTime)
-  {
-    DS18B20.requestTemperatures();
-    DSreqTime = millis();
-  }
-}
-
 void initDS18B20()
 {
-  if (my_OWpin == -1)
+  for (uint8_t p = 0; p < sizeof(OW_PINS); p++)
```

Comment:
this would perform a scanning of pins on every wake up which is a time/battery waste and potential runtime instability.
I don't see a reason why this is better than the initial action during setup and only happening at that supervised occasion.

Reply:
Fixed